components/ApiKeyUsageChart.js

```diff
@@ -0,0 +1,187 @@
+import { Key } from '@styled-icons/fa-solid/Key'
+import moment from 'moment'
+import { Component } from 'react'
+import { Button } from 'react-bootstrap'
+import {
+  XYPlot,
+  XAxis,
+  YAxis,
+  VerticalGridLines,
+  HorizontalGridLines,
+  VerticalRectSeries
+} from 'react-vis'
+import { withAuth } from 'use-auth0-hooks'
+
+/**
+ * Renders a chart showing API Key usage (requests over time) for a particular
+ * API key.
+ */
+class ApiKeyUsageChart extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      value: null
+    }
+  }
+
+  _clearValue = (data, event) => {
+    this.setState({value: null})
+  }
+
+  _getChartTitle = () => {
+    const { aggregatedView, isAdmin } = this.props
+    // Do not show chart title for non-admin users.
+    if (!isAdmin) return null
+    const defaultTitle = aggregatedView ? 'Total Requests' : 'Unknown Application'
+    const defaultUser = aggregatedView ? 'All users' : '[no user]'
```

Comment:
Add FIXMEs or something for these hardcoded values `Unknown Application`, `[no user]`.

Reply:
I see that these are used when there isn't an API User. Seems like we should still have some kind of mechanism to trace what these non-api user keys are all about.

Reply:
Right now, the "unknown" keys are the ones we've created outside of the flow of the otp-admin-ui (i.e., we created them manually in the AWS console and used them in otp-react-redux). I think it might be more appropriate to handle this on the server side (e.g., filtering out keys if they don't match user accounts), but there may be cases where we want to see these keys as well, so I'm inclined to leave it as is until we have a better sense of things down the road.